external/grpc/Kconfig.protocol

```diff
@@ -6,7 +6,7 @@
 config GRPC
 	bool "gRPC"
 	default n
-	depends on NET
-	depends on HAVE_CXX
+	select PROTOBUF
+	depends on NET && HAVE_CXX && LIBCXX
```

Comment:
Because LIBCXX is in HAVE_CXX conditional in Kconfig of libxx, you had better check only LIBCXX. Checking both is redundant.